### ISSUE TITLE
Codemirror loading fails in certain circumstances due to `innerCodeElem` reference

### ISSUE THREAD

**Issue by ssddanbrown** [BookStackApp/BookStack#4917]
Error thrown when viewing page:

```
Uncaught (in promise) TypeError: innerCodeElem is null
    highlightElem index.mjs:58
    highlight index.mjs:93
    setup page-display.js:38
    promise callback*setup page-display.js:38
    initComponent components.js:95
    init components.js:122
    <anonymous> app.js:37
```

Can reproduce with the following in the MD editor: (ignoring backslashes)

```
\```
   blank format
\```
```

Reported by [Reddit user BaccanoMob here](https://www.reddit.com/r/BookStack/comments/1bpqga0/how_do_i_make_the_code_block_with_line_numbers/).

**Comment by guillerg86** [comment]
Seems also failed the "copy" shortcut on code block at top right corner


![codeblock-nocopy-shortcut](https://github.com/BookStackApp/BookStack/assets/10212751/4149c699-2c57-4af8-85fa-08e5488b1a32)

**Comment by ssddanbrown** [comment]
Closed within 58f6219cb3cf937fef59d03d4420de1ba64df436, to be part of the next patch release.
